processor/src/services/converters/create-payment.converter.ts

```diff
@@ -36,12 +36,16 @@ export class CreatePaymentConverter {
         return this.populateAdditionalCardData();
       case 'klarna':
       case 'klarna_paynow':
-      case 'klarna_b2b':
       case 'klarna_account': {
         return {
           lineItems: mapCoCoCartItemsToAdyenLineItems(cart),
         };
       }
+      case 'klarna_b2b': {
+        return {
+          ...this.populateKlarnaB2BData(cart),
+        };
```

Comment:
unnecessary spread operator
```suggestion
        return this.populateKlarnaB2BData(cart);
```